packages/cli/src/lib/snapshot/snapshotFacade.ts

```diff
@@ -12,9 +12,11 @@ import {SynchronizationPlan} from './synchronization/synchronizationPlan';
 export class SnapshotFacade {
   public constructor(private snapshot: Snapshot, private cfg: Configuration) {}
 
-  public async tryAutomaticSynchronization() {
+  public async tryAutomaticSynchronization(skipPrompt = false) {
```

Comment:
Nitpick, but I would just reverse that to prompt = true, so there would be no "double negative".

And in the code: 

```
if(prompt) {
   await this.waitForConfirmation();
}
[...]
```